geonode/static/geonode/js/search/search.js

```diff
@@ -421,7 +421,7 @@
     * and pushes/removes the value of the element from the query object
     */
     $scope.multiple_choice_listener = function($event){
-      var element = $(event.currentTarget);
+      var element = $($event.currentTarget);
```

Comment:
Probably just a typo here, I add `$` sign to follow conventions with other functions.